x-pack/plugins/security_solution/public/management/cypress/e2e/artifacts.cy.ts

```diff
@@ -61,14 +61,19 @@ const loadEndpointDataForEventFiltersIfNeeded = () => {
 };
 
 const runAction = (action: FormAction) => {
+  let element;
+  if (action.customSelector) {
+    element = cy.get(action.customSelector);
+  } else {
+    element = cy.getBySel(action.selector || '');
+  }
+
   if (action.type === 'click') {
-    cy.getBySel(action.selector).click();
+    element.click();
   } else if (action.type === 'input') {
-    cy.getBySel(action.selector).type(action.value || '');
+    element.type(action.value || '');
   } else if (action.type === 'clear') {
-    cy.getBySel(action.selector).clear();
-  } else if (action.type === 'customClick') {
-    cy.get(action.selector).click();
+    element.clear();
   } else if (action.type === 'removeItem') {
     cy.get(`[data-test-subj="${action.selector}"] > span > button`).click();
```

Comment:
always those exceptions! :D

do you think this would work like this?
`element.find('span > button').click()`

or can this be seen as a `customSelector` with a `click` action?